package.json

```diff
@@ -22,6 +24,8 @@
     "react-lottie": "^1.2.3",
     "react-router-dom": "^4.3.1",
     "react-scripts": "^2.1.8",
+    "react-scripts-ts": "^4.0.8",
```

Comment:
`"react-scripts-ts"` you may have resolved a merge conflict where I removed this package. I migrated our app to CRA v2 so we can drop this.